_pkgdown.yml

```diff
@@ -56,8 +60,8 @@ reference:
 
 navbar:
   structure:
-    left: [getstarted, reference, articles, news]
-    right: [search, slack, newissue, github]
+    left: [getstarted, reference, articles, news, versions]
```

Comment:
Do we need to define the `versions` entry?

Reply:
When I build the website I get:
![image](https://github.com/pharmaverse/admiralonco/assets/80953585/af3f761d-66a3-4769-b9b9-b763e61609b4)
I would expect a "Versions" entry to the right of "Changelog".

Reply:
I think this was a leftover from the example with the history link in two places and could be removed